content/python/concepts/operators/operators.md

```diff
@@ -15,64 +15,62 @@ CatalogContent:
   - 'paths/data-science'
 ---
 
-Operators are used to perform operations on variables and values.
+Operators are used to perform various operations on variables and values. The standard arithmetic and assignment operators are the most familiar.
 
-In the example below, we use the `+` operator to add together two values:
+## Syntax
+
+The following code snippet uses an arithmetic operator for multiplication, `*`, to calculate the product of two numbers, along with the assignment operator, `=`, to store the result in the variable `x`.
 
 ```py
-print(10 + 5)
+# Store five times three in variable x
+x = 5 * 3
```

Comment:
I have an after-thought which is to make this a bit more reusable.

> The following code snippet assigns `my_variable` to the value of `num1` and `num2` with some arithmetic operator acting on them. For example, if the operator represented an asterisk, `*`,  the `my_variable` would be assigned a value of `num1 * num2`.

```
my_variable = num1 operator num2... and so on
```

Reply:
Maybe we can mention order of operations as well

Reply:
@yangc95 did both.